lib/components/ThingComment.js

```diff
@@ -55,7 +55,8 @@ const ThingComment = ({
   created_utc,
   showCommentForm,
   onCloseCommentForm,
-  highlightcomment
+  highlightcomment,
+  userAccessTime
```

Comment:
Instead of passing the userAcessTime here can we make a new boolean prop like `isCommentNew` and implement the logic to control its value out of snew?

Reply:
The idea of the `highlightcomment` is exactly to move logic out of snew, it wikk work for both permalink, new comments, or any kind of comments that should be highlighted. On pgui would be something like this: `highlightcomment: isCommentPermalink || isCommentNew`

Reply:
I advise you to adapt the logic on pgui and use the existing prop, @victorgcramos